itools-packager/src/main/resources/itools

```diff
@@ -44,8 +44,11 @@ done
 options+=" -Dpowsybl.config.dirs=$powsybl_config_dirs"
 [ -n "$powsybl_config_name" ] && options+=" -Dpowsybl.config.name=$powsybl_config_name"
 options+=" -Dlogback.configurationFile="
-[ -f "$powsybl_config_dirs/logback-itools.xml" ] && options+="$powsybl_config_dirs" || options+="$installDir/etc"
-options+="/logback-itools.xml"
+logFile="$installDir/etc/logback-itools.xml"
+for dir in ${powsybl_config_dirs//:/ }; do
+    [ -f "$dir/logback-itools.xml" ] && logFile="$dir/logback-itools.xml"
+done
+options+="$logFile"
```

Comment:
Add quotes to handle spaces in paths

Reply:
It was not really practical to do it that way so I used an array.
I also modified `options` as an array as the java command would fail otherwise if `powsybl_config_dirs` contained spaces.